prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -770,6 +781,16 @@ class ActionHistory(
         }
     }
 
+    /**
+     *  Sends any FHIR queue messages to the appropriate queue
+     */
+    fun queueFhirMessages(workflowEngine: WorkflowEngine) {
```

Comment:
Can we add a happy path test for this? Some parts of this file are a beast but I think it should be pretty easy to add in some messages with `trackFhirMessage` and then verify they get processed here.